a3m/client/clientScripts/create_mets_v2.py

```diff
@@ -429,7 +429,8 @@ def create_premis_object(fileUUID):
     creatingApplication = etree.Element(ns.premisBNS + "creatingApplication")
     etree.SubElement(
         creatingApplication, ns.premisBNS + "dateCreatedByApplication"
-    ).text = f.modificationtime.strftime("%Y-%m-%d")
+    ).text = f.modificationtime.strftime("%Y-%m-%dT%H:%M:%S%z")
```

Comment:
In Archivematica, the decision was to use `Z` instead of `z` based on the [EDTF specification](https://www.loc.gov/standards/datetime/). We should do the same here.

See https://github.com/artefactual/archivematica/pull/1705 for more details.